aws_doc_sdk_examples_tools/metadata.py

```diff
@@ -151,6 +149,30 @@ class Example:
     synopsis_list: List[str] = field(default_factory=list)
     source_key: Optional[str] = field(default=None)
 
+    def fill_title(self, categories: Dict[str, Category], service, action):
```

Comment:
I don't love `fill_title` as the name of this function, given that it fills more than the title.

Reply:
Done.